src/Onboarding/Welcome.tsx

```diff
@@ -18,87 +13,74 @@ import { Spacing, Colors, Typography, Outlines } from "../styles"
 import { Screens, OnboardingScreens, useStatusBarEffect } from "../navigation"
 
 const Welcome: FunctionComponent = () => {
-  useStatusBarEffect("light-content")
   const navigation = useNavigation()
   const {
     t,
     i18n: { language: localeCode },
   } = useTranslation()
   const languageName = getLocalNames()[localeCode]
   const appName = env.IN_APP_NAME || "PathCheck"
+  useStatusBarEffect("dark-content")
 
   return (
-    <ImageBackground
-      source={Images.BlueGradientBackground}
-      style={style.backgroundImage}
-    >
-      <View style={style.container}>
-        <View style={style.headerContainer}>
-          <TouchableOpacity
-            onPress={() => navigation.navigate(Screens.LanguageSelection)}
-            style={style.languageSelector}
-          >
-            <GlobalText style={style.languageSelectorText}>
-              {languageName}
-            </GlobalText>
-          </TouchableOpacity>
-        </View>
-        <View>
-          <GlobalText style={style.mainText}>
-            {t("label.launch_screen1_header")}
-          </GlobalText>
-          <GlobalText style={style.mainText}>{appName}</GlobalText>
-        </View>
-        <View style={style.footerContainer}>
-          <Button
-            invert
-            label={t("label.launch_get_started")}
-            onPress={() => navigation.navigate(OnboardingScreens.Introduction)}
-          />
-        </View>
+    <View style={style.container}>
+      <TouchableOpacity
+        onPress={() => navigation.navigate(Screens.LanguageSelection)}
+        style={style.languageButtonContainer}
+      >
+        <GlobalText style={style.languageButtonText}>{languageName}</GlobalText>
+      </TouchableOpacity>
+      <View>
+        <Image
+          source={Images.MinnesotaMap}
```

Comment:
Does this need to be conditional on the build? we probably dont want other HAs to have this background image?

Reply:
Yeah it does, I was going to cut a card for this work. Thoughts?